ibrahim/codepen.html

```diff
@@ -0,0 +1,170 @@
+    <li><a href="#">Publications</a></li>
+    <li><a href="#">Contact</a></li>
+  </ul>
+</nav>
+
+<!-- Header Image -->
+<img src="http://angeliqueweger.com/temp/gd2-images/gallery3.jpg" alt="">
+
+<div class="body">
+
+  <div class="text">
+    <ul class="text-grid">
+
+      <li>
+        <h2>Dance of Conversation</h2>
+        <p>Friday March 6, 2020 - Saturday, March 28, 2020
+          Opening Reception: Saturday, March 14, 2020</p>
+      </li>
+
+      <li>
+        <h2>Featured Artists</h2>
+        <p>Shelley Amsel, Jude Asher, Toni Berger, Mary Bickford, Joel Cohen, Diane Dennis, Landis Expandis, Jackie Mintz, Michael Tice, and Carlos Walker</p>
+      </li>
+
+    </ul>
+  </div>
+
+  <div class="main-content">
+    <ul class="art-list">
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=1zqe-3V6Yr-X5IV_bUX1imhK2b98YZhnN" alt="">
+          <figcaption>Shelley Amsel</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=1JauRZ9G0dB80UgPzjnyUoRQ3AbHRKn7_" alt="">
+          <figcaption>Jude Asher</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=11ZIHfAkVc-kXKevqW4jrSDrsFxcbu3RE" alt="">
+          <figcaption>Toni Berger</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=1JbS934_409Zm-PQBQ6-E9vVlAxr-7HBb" alt="">
+          <figcaption>Mary Bickford</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=1n7V5y6j1B2XdSTpvMPS2sR3J_Ko5_V00" alt="">
+          <figcaption>Joel Cohen</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=1TN-A7EqFaqhMVrSAZA82KEHBReXLrgw3" alt="">
+          <figcaption>Jackie Mintz</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=13eB7vleT7IfpBpRrtaYO65j0ER1Y1jGb" alt="">
+          <figcaption>Landis Expandis</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+
+          <img src="https://drive.google.com/uc?id=1Kd5E9hQl5yos1_pqJuRAoIArjSLIux4l" alt="">
+          <figcaption>Michael Tice</figcaption>
+
+        </figure>
+      </li>
+
+      <li>
+        <figure>
+          <div class="img-cont">
+            <img src="https://drive.google.com/uc?id=15xqhP2IgthSbpCFF1jkyh-IrLFlG--oJ" alt="">
+            <figcaption>Carlos Walker</figcaption>
+          </div>
+        </figure>
+      </li>
+
+    </ul>
+
+    <div class="text">
+      <ul class="text-grid">
+        <li>
+          <h2>Curated by : Jude Asher</h2>
+          <p>Featuring an artist talk led by Sheila Gaskinsr</p>
+        </li>
+      </ul>
+    </div>
+
+  </div>
+
+  <div class="footer">
```

Comment:
Instead of a meaningless `div` with a classname, consider making this a `footer` element instead?